File: InquirerDLL/Output/Survey/Statistics.cs
using System.Collections.Generic;

namespace InquirerDLL.Output.Survey
{
    public class Statistics
    {
        public List<Question> Questions { get; set; }


        public class Question
        {
            public int Id { get; set; }
            public string Title { get; set; }
            public int TypeId { get; set; }
            public int Maximum { get; set; }
            public List<Option> Options { get; set; }
        }

        public class Option
        {
            public int? Id { get; set; }
            public string Label { get; set; }
            public string Value { get; set; }
            public int Quantity { get; set; }
            public List<Answer> Answers { get; set; }
        }

        public class Answer
        {
            public string Content { get; set; }
            public int Quantity { get; set; }
        }

    }

}

File: Inquirer/wwwroot/vue/root/survey/statistics.js
class Question {
    constructor(key, id, title, typeId, maximum, options) {
        this.key = key;
        this.id = id;
        this.title = title;
        this.typeId = typeId;
        this.maximum = maximum;
        this.options = options;
    }
}

var application = new Vue({
    el: "#page-root",
    data: {
        survey: model.survey,
        user: model.user,
        statistics: model.statistics,
        questions: []
    },
    created: function () {
        for (var i = 0; i < this.statistics.questions.length; i++) {
            var q = this.statistics.questions[i];
            this.questions.push(new Question(i, q.id, q.title, q.typeId, q.maximum, q.options));
        }
        $("#model").detach();
    },
    methods: {
        Format: function (date, format) {
            return $.format.date(date, format);
        }
    }
});

